key.js

```diff
@@ -1 +1,2 @@
-var APP_KEY = 'b891af3a0d98ce8944cb61daf088bc8c';
\ No newline at end of file
+var APP_KEY = 'b891af3a0d98ce8944cb61daf088bc8c';
+var AnalyticsCode = 'Your Analytics Code - UA-XXXX-X'
```

Comment:
Can you see if you can use both these variables as `const` instead? I should have seen this before, since they don't change const will be a better alternative.

Reply:
Done